geometry/SettingsViewController.swift

```diff
@@ -31,25 +29,32 @@ class SettingsViewController: UIViewController
     @IBOutlet weak var labelGreen: UILabel!
     @IBOutlet weak var labelBlue: UILabel!
     
-    
+    @IBAction func close(sender: AnyObject) {
+        //dismissViewControllerAnimated(true, completion: nil)
+        let rColor = Float(redСolor.value / 255)
+        let gColor = Float(greenСolor.value / 255)
+        let bColor = Float(blueСolor.value / 255)
+        delegate?.fillSettingsViewControllerDelegate(rColor,gColor: gColor,bColor: bColor)
+        navigationController?.popViewControllerAnimated(true)
+    }
     
     
     @IBAction func colorChanged(sender: UISlider) {
```

Comment:
Does this happen on each change of a slider? The color should be passed back once a user finishes adjusting the settings and clicks Done button.